JASP-Engine/JASP/R/reliabilityAnalysisBayesian.R

```diff
@@ -416,22 +416,22 @@ reliabilityBayesian <- function(jaspResults, dataset, options) {
   for (i in idxSelected) {
     if (estimators[i] %in% coefficients) {
       if (estimators[i] == "Item-rest correlation") { # no item deleted for item rest cor
-        itemTable$addColumnInfo(name = paste0("postMean", i), title = "Posterior Mean", type = "number", 
+        itemTable$addColumnInfo(name = paste0("postMean", i), title = gettext("Posterior Mean"), type = "number", 
                                 overtitle = gettext("Item-rest correlation"))
-        itemTable$addColumnInfo(name = paste0("lower", i), title = paste0("Lower ", cred, "%"), type = "number", 
+        itemTable$addColumnInfo(name = paste0("lower", i), title = gettextf("Lower %s %%", cred), type = "number", 
```

Comment:
```suggestion
        itemTable$addColumnInfo(name = paste0("lower", i), title = gettextf("Lower %s%% CI", cred), type = "number", 
```

This is only a minor suggestion, as we are not entirely consistent with this in JASP anyway. However, I would at least delete the space between `cred` and the percentage sign (e.g., to display `95%` and not `95 %`)